src/ode_solver/relaxation_runge_kutta/algebraic_rrk_ode_solver.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __ENERGY_RRK_ODESOLVER_H__
+#define __ENERGY_RRK_ODESOLVER_H__
+
+#include "dg/dg_base.hpp"
+#include "rrk_ode_solver_base.h"
+#include "ode_solver/ode_solver_base.h"
+#include "ode_solver/runge_kutta_ode_solver.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// Relaxation Runge-Kutta ODE solver, calculating the relaxation parameter as in Ketcheson 2019
+/** "Relaxation Runge-Kutta Methods: Conservation and Stability for Inner-Product Norms" 
+ */
+#if PHILIP_DIM==1
+template <int dim, typename real, typename MeshType = dealii::Triangulation<dim>>
+#else
+template <int dim, typename real, typename MeshType = dealii::parallel::distributed::Triangulation<dim>>
+#endif
+class AlgebraicRRKODESolver: public RRKODESolverBase<dim, real, MeshType>
+{
+public:
+    /// Default constructor that will set the constants.
+    explicit AlgebraicRRKODESolver(
+            std::shared_ptr<RKTableauBase<dim,real,MeshType>> rk_tableau_input);
+
+protected:
+
+    /// Compute relaxation parameter explicitly (i.e. if energy is the entropy variable)
+    /// See Ketcheson 2019, Eq. 2.4
```

Comment:
same comment for the full reference

Reply:
I think it's sufficient to put the full reference only at the top of the .h file. I have also added the full reference to the root-solving equivalent.